composer.json

```diff
@@ -10,7 +10,7 @@
     "require": {
         "ezsystems/platform-ui-assets-bundle": "^3.1.0",
         "ezsystems/repository-forms": "^1.2",
-        "ezsystems/ezpublish-kernel": "^6.6",
+        "ezsystems/ezpublish-kernel": "~6.7.8 || ^6.13.2",
```

Comment:
you can set it to `~6.7.8@dev || ^6.13.2@dev`, we'll ideally have to cleanup after release, but otherwise this will break travis on merge